### DISCUSSION TITLE
HALMET boards now available at shop.hatlabs.fi

### DISCUSSION THREAD

**Discussion by mairas** [hatlabs/discussions#81] [General]
Hey all, HALMET (Hat Labs Marine Engine & Tank interface) is now available at [shop.hatlabs.fi](https://shop.hatlabs.fi/products/halmet)!

HALMET lets you interface common engines and tank senders with your Signal K or NMEA 2000 network. Among other things, you can:

- Measure RPM from alternator outputs or Hall effect sensors
- Connect tank level, engine pressure and other resistive senders
- Connect alarms
- Measure voltages

HALMET has four analog and four digital signal inputs. Both the digital and the analog inputs have a voltage range of 0-32 volts. They also feature galvanic isolation from the main board, solving any ground loop issues once and for all. Analog inputs can measure sender resistance either actively or in parallel with existing gauges in a passive voltage measurement mode.

I still need to finish the sales packaging and the quick start guide, so shipments will begin after 2023-12-26. Initial documentation will also be prepared around that time.

For an early bird discount, follow [this link](https://shop.hatlabs.fi/discount/HALMETSK?redirect=%2Fproducts%2Fhalmet) (valid until 2023-12-28)!

![PC210009](https://github.com/hatlabs/discussions/assets/407010/66bc113a-c0a3-4465-8bc3-b30dcdfa72f8)

**Comment by psm12** [comment]
Hi Matti,Your early bird link appears to be broken. PaulOn 21 Dec 2023, at 18:54, Matti Airas ***@***.***> wrote:﻿
Hey all, HALMET (Hat Labs Marine Engine & Tank interface) is now available at shop.hatlabs.fi!
HALMET lets you interface common engines and tank senders with your Signal K or NMEA 2000 network. Among other things, you can:

Measure RPM from alternator outputs or Hall effect sensors
Connect tank level, engine pressure and other resistive senders
Connect alarms
Measure voltages

HALMET has four analog and four digital signal inputs. Both the digital and the analog inputs have a voltage range of 0-32 volts. They also feature galvanic isolation from the main board, solving any ground loop issues once and for all. Analog inputs can measure sender resistance either actively or in parallel with existing gauges in a passive voltage measurement mode.
I still need to finish the sales packaging and the quick start guide, so shipments will begin after 2023-12-26. Initial documentation will also be prepared around that time.
For an early bird discount, follow this link (valid until 2023-12-28)!
PC210009.JPG (view on web)

**Reply by mairas** [reply]
Indeed! Thanks for the heads-up.

Should be fixed now. And if that doesn't work, use the discount code HALMETSK at checkout.

**Comment by psm12** [comment]
Hi Matti,
I’ve ordered a HALMET. I have a couple of questions, I used shopify .?  to create an account. Is this safe? Also the shipping to Uk is quite expensive which unfortunately takes the edge off your pricing.
Paul


Sent from Yahoo Mail for iPad


On Friday, December 22, 2023, 13:09, Matti Airas ***@***.***> wrote:




Indeed! Thanks for the heads-up.

Should be fixed now. And if that doesn't work, use the discount code HALMETSK at checkout.

**Reply by mairas** [reply]
Yes, Shopify accounts are safe. Merchants do not have access to any of you critical information (credit card numbers etc).

Unfortunately the shipping costs to the UK have become much higher since Brexit. I encourage customers to make bigger one-time purchases - there's a limit banner shown at the top of every shop page above which you get free shipping.

**Comment by psm12** [comment]
Thanks Matti,
I know full documentation for HALMET is still incomplete. Are there schematics available?
Thanks,
Paul
    On Saturday, 23 December 2023 at 15:57:45 GMT, Matti Airas ***@***.***> wrote:  
 
 


Yes, Shopify accounts are safe. Merchants do not have access to any of you critical information (credit card numbers etc).

Unfortunately the shipping costs to the UK have become much higher since Brexit. I encourage customers to make bigger one-time purchases - there's a limit banner shown at the top of every shop page above which you get free shipping.

**Comment by addboy58** [comment]
I’m I right to assume that the alternator line from the analog wire harness will work in the D1 inputted? You Ned pos and neg from the harness? Any examples on how to do temp and or pile pressure?

**Reply by big-col** [reply]
maybe create a new thread for this question?  it might get lost/unnoticed here
